.env.example

```diff
@@ -1,3 +1,12 @@
+# Set the port to run Vite Developemnt Server
+PORT=3000
+
+# Use Host Name for Vite
+HOST_NAME=localhost
+
+# Expose Vite to Network
+EXPOSE_NETWORK=false
```

Comment:
that .env var is self explain. If user want to expose it on Network set it true . 
It is set as false because let says if u don't want to expose it on network so user have control over it instead by default exposing it **HARD CODED** that is aim of that variable so in future user won't have to  **rr-edit vite.config.js** and set it false.

Reply:
I will set it as True in push changes in few minutes

Reply:
I already set it as true. Okay. I will set it as false now.

please point the docs where it need to mention so i will add it there

Reply:
Can add explanation for variables at https://github.com/milancommunity/Milan/blob/main/docs/FrontendSetup.md#setting-up-the-env-file